test/parallel/test-x509-escaping.js

```diff
@@ -502,3 +502,35 @@ const { hasOpenSSL3 } = common;
     }, common.mustCall());
   }));
 }
+
+// certificateExtensions test
+{
+  const pem = fixtures.readSync('x509-certificate-extensions.pem', 'utf8');
```

Comment:
I think you forgot to add this file in your commit?

Reply:
@mscdex I missed it, sorry, do you have any suggestions where I should put this file? I wonder

Reply:
I would suggest test/fixtures/keys and also adding the certificate's generation steps into the Makefile in the same folder.

Reply:
@panva Thank you very much for your reply. I wonder if there is any certificate creation document, I have a certificate in my first development on this subject, but I created a random certificate for testing.

Reply:
I tried a few steps, I'll be sharing it with a commit soon 🙏  @panva @mscdex